### DISCUSSION TITLE
make is not an internal or external command

### DISCUSSION THREAD

**Discussion by sera-1994** [elyra-ai/elyra#2852] [Q&A]
In development environment，when I use command `make`  to list commands，it tells me make is not an internal or external command. And I want to know how to use `make` .

**Comment by ptitzler** [comment]
Which platform are you on?

**Reply by kevin-bates** [reply]
"_not an internal or external command_" implies a Windows platform.  Since we do not run CI on Windows, I'm not certain how solid our build is going to be (although there's no statement that we _don't_ support Windows development or runtime).

At a minimum, we should probably list [`GNU Make`](https://www.gnu.org/software/make/) in our [prerequisites](https://elyra.readthedocs.io/en/latest/developer_guide/development-workflow.html#requirements), similar to what we do in [Enterprise Gateway](https://jupyter-enterprise-gateway.readthedocs.io/en/latest/contributors/devinstall.html#prerequisites).  I would try downloading `make` ([here's some related information for Windows](http://gnuwin32.sourceforge.net/packages/make.htm) that essentially points back to the GNU make page) and see how far you get (although you'll be better off developing in a Unix-variant OS).
